src/include/connection.h

```diff
@@ -193,6 +193,9 @@ struct __wt_disaggregated_storage {
     /* To copy at the next checkpoint. */
     TAILQ_HEAD(__wt_disagg_copy_metadata_qh, __wt_disagg_copy_metadata) copy_metadata_qh;
     WT_SPINLOCK copy_metadata_lock;
+
+    bool internal_page_delta;
```

Comment:
Should this be inside the layered_table_manager?

Reply:
I can move it in here if that makes more sense.

Reply:
not really actually. I think it should be inside disaggregated_storage.

Reply:
not really. I think it should be inside disaggregated_storage